dashboard/app/controllers/api/v1/json_api_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::JsonApiController < ApplicationController
+  layout false
+
+  # Don't bother redirecting to login when denying access to the JSON APIs
+  rescue_from CanCan::AccessDenied do
+    head :forbidden
+  end
```

Comment:
Extracted common controller behavior for our new JSON APIs:

- We're not rendering views, don't use the site layout for 404s, etc.
- When a logged-out user is denied access just return a 403, don't redirect to the login form.

I'm only using these in the new section APIs so far, but we could certainly reuse this elsewhere (in a future PR).